pkg/storage/factory.go

```diff
@@ -44,16 +45,17 @@ type StoreLimits interface {
 
 // Config chooses which storage client to use.
 type Config struct {
-	AWSStorageConfig       aws.StorageConfig       `yaml:"aws"`
-	AzureStorageConfig     azure.BlobStorageConfig `yaml:"azure"`
-	GCPStorageConfig       gcp.Config              `yaml:"bigtable"`
-	GCSConfig              gcp.GCSConfig           `yaml:"gcs"`
-	CassandraStorageConfig cassandra.Config        `yaml:"cassandra"`
-	BoltDBConfig           local.BoltDBConfig      `yaml:"boltdb"`
-	FSConfig               local.FSConfig          `yaml:"filesystem"`
-	Swift                  openstack.SwiftConfig   `yaml:"swift"`
-	GrpcConfig             grpc.Config             `yaml:"grpc_store"`
-	Hedging                hedging.Config          `yaml:"hedging"`
+	AWSStorageConfig       aws.StorageConfig         `yaml:"aws"`
+	AzureStorageConfig     azure.BlobStorageConfig   `yaml:"azure"`
+	BosStorageConfig       baidubce.BosStorageConfig `yaml:"bos"`
```

Comment:
we should capitalize BOS here or use BCE (if that's the acronym for baidu cloud)

Reply:
tks,all `Bos` change to `BOS`.